index.js

```diff
@@ -0,0 +1,10 @@
+const openMenu = document.querySelector('.menu-icon button');
+const closeMenu = document.querySelector('.mobile-menu button');
+
+openMenu.addEventListener('click', () => {
+  document.querySelector('.mobile-menu').classList.add('show');
+});
+
+closeMenu.addEventListener('click', () => {
+  document.querySelector('.mobile-menu').classList.remove('show');
+});
```

Comment:
- [ ] Kindly as said in the requirements `When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`, when I click on any option the mobile menu disappears Kindly close it.

Reply:
@keroloussamy , I must have missed that. I have added that functionality. Please review again